doc/protocol.md

```diff
@@ -77,8 +77,8 @@ open until the "init" message has been received.
 
 The following fields are defined:
 
- * "version": The version of the protocol. Currently zero, and unstable.
- * "capabilities": An array of capability strings
+ * "version": The version of the protocol. Currently 1, and stable.
```

Comment:
This document is that documentation. We'll need to make notes in this document when we add something, and note the capabilities that are required for it to be present.